### PR TITLE
Added a couple of Tips to the Lifecylce Events section

### PR DESCRIPTION
**Pull request by jdh2550**
Added the concept of a "tip" to provide extra information to the n@@b developer (like me!)

### CODE REVIEW

**Comment by jdh2550** [comment]
Note - this adds the concept of a "Tip" to the README.  It could be styled better - and maybe even collapsible so as not to pollute the main flow of the existing document?

**Comment by jorgebucaran** [comment]
@jdh2550 Yeah, I agree. For starters I would remove the _italics_. I am not sure about calling them tips either. Can you condense both "tips" into a new paragraph?

File: README.md [hunk 1]
```diff
@@ -393,6 +393,8 @@ export const Jumbotron = ({ text }) => (
 
 You can be notified when elements managed by the virtual DOM are created, updated or removed via lifecycle events. Use them for animation, data fetching, wrapping third party libraries, cleaning up resources, etc.
 
+Note that lifecycle events are attached to DOM nodes not to components.  Consider adding a key to ensure that the event is attached to a specific DOM item, rather than a recycled one.
```
Comment (zaceno) [inline README.md]: “DOM nodes” should be virtual-DOM nodes, and “DOM items” should be “DOM elements”

Comment (zaceno) [reply README.md]: Other than that, I like it!

File: README.md [hunk 2]
```diff
@@ -393,7 +393,7 @@ export const Jumbotron = ({ text }) => (
 
 You can be notified when elements managed by the virtual DOM are created, updated or removed via lifecycle events. Use them for animation, data fetching, wrapping third party libraries, cleaning up resources, etc.
 
-Note that lifecycle events are attached to DOM nodes not to components.  Consider adding a key to ensure that the event is attached to a specific DOM item, rather than a recycled one.
+Note that lifecycle events are attached to virtual-DOM nodes not to components.  Consider adding a key to ensure that the event is attached to a specific DOM elements, rather than a recycled one.
```
Comment (jorgebucaran) [inline README.md]: @jdh2550 "virtual DOM"

Also, there's two white spaces before "  Consider".

**Comment by jorgebucaran** [comment]
🎉

**Pull request merged**